pages/dkp/kommander/2.3/clusters/custom-domain-certificate/configuration/index.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: layout.pug
+navigationTitle: Configuration 
+title: Configure a custom domain and certificate for your cluster
+menuWeight: 20
+excerpt: Configure a custom domain and certificate in the Management and any Managed or Attached Clusters
+beta: false
+enterprise: false
+---
+
+This section describes how to enable custom domains and certificates in your **Management and any Managed or Attached** clusters after the installation of DKP. If you want to set up a custom domain and certificate for your Management Cluster **during the installation**, refer to the [Customize a domain and certificate during installation][management] documentation.
+
+To customize the domain or certificate for a specific cluster after the installation of DKP, adapt or create an API `yaml` that will allow Kommander to implement the established adjustments on top of the default or any previous configuration you have made.
+
+Note that the `Issuer` object must be created on the cluster where you want to customize the configuration. In the Management cluster, both the `KommanderCluster` and `Issuer` objects are on the same cluster. In Managed and Attached clusters, the `KommanderCluster` object is stored on Management cluster, and the `Issuer` object is on the Managed or Attached cluster.
```

Comment:
@mhrabovcin the info you provided seemed important, so I added it here. But now i am wondering how this differentiates from the note we have in line 17?